drivers/hmis/app/graphql/types/base_audit_event.rb

```diff
@@ -54,6 +59,16 @@ def self.build(node_class, field_permissions: nil, excluded_keys: nil, transform
       arg :user, [ID]
     end
 
+    def client_name
+      client = load_ar_association(object, :hmis_client)
+      client&.full_name
+    end
+
+    def project_name
+      project = load_ar_association(object, :hmis_project)
+      project&.ProjectName
```

Comment:
We have all the hud column names aliased to snake case, so we usually use those in the code
```suggestion
      load_ar_association(object, :hmis_project)&.project_name
```